.github/workflows/CI.yml

```diff
@@ -181,4 +181,6 @@ jobs:
         pytest -vx tests/test_num_params.py
         pytest -vx tests/test_param_shapes.py
         pytest -vx tests/test_param_types.py
-        pytest -vx tests/test_ssim.py
\ No newline at end of file
+        pytest -vx tests/test_ssim.py
+        pytest -vx tests/submission_runner_test.py
+        pytest -vx tests/test_baselines.py
```

Comment:
will this download the MNIST data? that'll be a bit slow but probably is OK

Reply:
It took about 4 minutes 37 seconds to run all 12 baseline tests I enabled so I think it is ok too.